library/java/io/envoyproxy/envoymobile/engine/BUILD

```diff
@@ -31,6 +31,7 @@ java_library(
         "EnvoyNativeResourceRegistry.java",
         "EnvoyNativeResourceReleaser.java",
         "EnvoyNativeResourceWrapper.java",
+        "EnvoyStreamIntelImpl.java",
```

Comment:
Curious on the departure from StreamInfo. Intended for a clear division from the Envoy interface?